feat: added new component Card

eakansh125 opened the pull request.

Commit:
feat: added new component Card

Naman-Saxena1 commented on card.html:
```diff
@@ -0,0 +1,267 @@
+                                <div class="card-lowerRight">
+                                    <img src="https://img.icons8.com/ios/18/000000/like--v1.png" />
+                                    <img src="https://img.icons8.com/material/18/000000/share--v1.png" />
+                                    <img src="https://img.icons8.com/material/18/000000/menu-2--v1.png" />
```
img with alt attribute is good practice.

Naman-Saxena1 commented on style.css:
```diff
@@ -365,4 +382,131 @@ main{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+} */
```
You can remove commented code if not needed.

Naman-Saxena1 reviewed (changes requested):
Your Card with badge has badge on icons, badge should be on the card itself.

swstica commented:
You can remove the commented code, the rest looks good.

rahulk31 commented:
Hey Eakansh, a small UI suggestion:

<img width="508" alt="image" src="https://user-images.githubusercontent.com/39650528/153232764-7c873940-6829-4323-9fd8-adc093eadd16.png">

You can either make the icons a bit larger or the badges bit small so that both contrasts well in size. You can plan to do so in next iteration, upto you.

veereshbv04 commented on style.css:
```diff
@@ -365,4 +382,131 @@ main{
+    border-radius: 5px;
+}
+
+.card-footer{
```
herat icon looks dull, increase the font-size

veereshbv04 commented on card.html:
```diff
@@ -0,0 +1,267 @@
+                                    </div>
+                                    <div class="badge-container">
+                                        <img src="https://img.icons8.com/ios-glyphs/18/000000/share--v1.png" />
+                                        <span>5</span>
```
decrease the size of badge, it looks good

mehulsatardekar commented on card.html:
```diff
@@ -0,0 +1,267 @@
+                    <div class="card">
+                        <div class="card-upper">
+                            <div class="image">
+                                <img src="img/stadium.jpg">
```
You can add lazy loading on images by adding the **loading="lazy"**  attribute to improve the performance.

mehulsatardekar commented on card.html:
```diff
@@ -0,0 +1,267 @@
+                    <div class="card horizontal">
+                        
+                            <div class="image">
+                                <img src="img/stadium.jpg">
```
Add lazy loading here too

mehulsatardekar commented on style.css:
```diff
@@ -365,4 +382,131 @@ main{
+    align-items: center;
+}
+
+.card-lowerRight>img{
```
try avoiding CSS combinators as much as possible it may create problems later on, suppose you want to make some images small and some big so another styling won't work here. instead, use CSS classes.

mehulsatardekar replied:
After the changes good to merge from my side.

Commit:
fix: fixed-commented code, badge, alt

eakansh125 commented:
Resolved:
1. Removed commented code
2. Added badge on the card instead of badge on icons
3. Added alt attribute to images
4. Planning to avoid combinators going forward
